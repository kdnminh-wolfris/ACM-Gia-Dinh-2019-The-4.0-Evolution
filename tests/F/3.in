8250 2204 3449 2533