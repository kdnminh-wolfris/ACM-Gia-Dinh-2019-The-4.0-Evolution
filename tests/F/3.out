6046 8250